auth/src/common.h

```diff
@@ -39,6 +39,9 @@ void InitPlatformAuth(AuthData* auth_data);
 // Platform-specific method to destroy the wrapped Auth class.
 void DestroyPlatformAuth(AuthData* auth_data);
 
+// Platform-specific method that causes a heartbeat to be logged.
```

Comment:
You can add a reference to go/firebase-platform-logging-design here

Reply:
Done.